lib/trace-object.js

```diff
@@ -12,8 +12,10 @@ exports.tracerOptions = tracerOptions;
 
 exports.sendTraceObject = function sendTraceObject(parentCtl) {
   if (cluster.isWorker) {
-    debug('starting trace object on worker %d', cluster.worker.id);
```

Comment:
@Setogit can you add this debug line back in and change it to something like:
```js
debug('attempting to start trace object on worker %d', cluster.worker.id);
```

That way the debug output will be helpful in determining the flow of thing even when tracing is unlicensed.